src/queries/index.js

```diff
@@ -31,6 +32,8 @@ export const getQuery = (query, filterOptions = {}) => {
     [QUERY_TYPES.EVENT_RECORDS]: filterOptions.showEventsFilter
       ? GET_EVENT_RECORDS_AND_CATEGORIES
       : GET_EVENT_RECORDS,
+    [QUERY_TYPES.GENERIC_ITEM]: GET_GENERIC_ITEMS,
```

Comment:
plural `GET_GENERIC_ITEMS` is correct here?

Reply:
added the singular query now